src/main/java/org/jenkinsci/plugins/ghprb/RegexUtil.java

```diff
@@ -20,7 +20,7 @@ private RegexUtil() {
 
     public static boolean find(Pattern pattern, String input, int timeoutMillis) {
         ExecutorService executor = Executors.newSingleThreadExecutor();
-        Future<Boolean> future = executor.submit(() -> pattern.matcher(input).find());
+        Future<Boolean> future = executor.submit(() -> pattern.matcher(new ThreadString(input)).matches());
```

Comment:
Why we revert back to`matches()`?

Reply:
The current `trigger-phrase` regular expressions work with `matches()` So I thought not to alter the default behaviour of the plugin. But if you have plans to amend them that require `find()` I can change it.

Reply:
No I don't have plans, was just wondering why change it. If `matches()` performs equally, I don't mind.